src/main.rs

```diff
@@ -20,6 +22,22 @@ fn main() -> Result<()> {
     }
     env_logger::init();
 
+    StdErr::deprecation_warning(&format!(
+        r#"{} {} {}
+
+You are using an outdated version of Wrangler (v{}).
+Please upgrade to v2 ({}).
```

Comment:
```suggestion
Please upgrade to the latest version ({}).
```

Reply:
The problem is that major versions aren't sacred, and we could make a wrangler v3 if we needed to introduce a breaking change.

So "the latest version" saves us from having to update it several times in the future.